### DISCUSSION TITLE
[Art competition] Buffer Overflow

### DISCUSSION THREAD

**Discussion by syunni** [prosyslab-classroom/cs348-information-security#400] [Art competition]
![DALL·E 2023-05-22 15 53 30 - I want to draw a scene where one student is studying  There are plenty of other study materials around  Meanwhile, above the student's head is a squar](https://github.com/prosyslab-classroom/cs348-information-security/assets/49365495/0f39a0bd-529f-4334-a46a-b389127693ad)

## Short Description
For me, "Buffer Overflow" means "I want to draw a scene where one student is studying. There are plenty of other study materials around. Meanwhile, above the student's head is a square box marked "Buffer".  The student "buffer overflows" and is unable to do other work properly.".

## Long Description (up to 3 sentences)
Buffer overflow means that other memory space is affected when data exceeding a certain size is written. If you study beyond the set storage capacity, you won't be able to learn any more and may negatively affect your other work. Let's study while taking enough breaks so that the buffer can be stored on disk, that is, long-term memory.

**Comment by KihongHeo** [comment]
Good. It would be nice if the aftermaths of buffer overflows could be expressed.
